HACKING.md

```diff
@@ -42,3 +44,37 @@ For a quick compile and debug cycle, we recommend iterating code using thorough
 unit tests before going through the full workflow described above.
 
 [1]: https://wiki.qemu.org/Documentation/Networking#User_Networking_.28SLIRP.29
+
+## Containers
+
+*osbuild-composer* and *osbuild-composer-worker* can be run using Docker
+containers. Building and starting containers is generally faster than building
+RPMs and installing them in a VM, so this method is more convenient for
+developing and testing changes quickly.
```

Comment:
Maybe also mention the possible downside (not testing the .spec file / integration into RHEL/Fedora), otherwise one might reasonably ask why we test in VMs at all.

Reply:
Good idea.  I'll add some caveats.

Reply:
Let me know what you think of the new text.